.bazelci/presubmit.yml

```diff
@@ -106,6 +106,16 @@ tasks:
     - "--test_tag_filters=-e2e,-examples,-fix-bazelci-ubuntu"
     test_targets:
     - "//..."
+  ubuntu1804_debug:
+    name: ubuntu1804_debug
+    platform: ubuntu1804
+    test_flags:
+    - "--define=VERBOSE_LOGS=1"
```

Comment:
hmm, I'm not sure it's worth adding CI load for this. let's just turn on these options in all our builds? I think spammy is probably good so we can understand CI failures for all jobs?

Reply:
Since `- "--define=DEBUG=1"` I think we need to run all tests without it set and with it set. No other way to know if the rules that change their output based on this break things.